.travis.yml

```diff
@@ -1,4 +1,4 @@
-if: branch = master OR tag IS present
+if: branch = publish
```

Comment:
Only run the Travis build (which just runs the publish to S3 script) from the `publish` branch. This branch doesn't exist yet. If we ever need to update a template for an older CLI, we can create the branch and use it to publish changes to legacy templates.